api/tests/core/users/test_api.py

```diff
@@ -141,7 +141,7 @@ def test_get_user_with_valid_token_wrong_type(self):
 
         users_factories.TokenFactory(user=user, value=self.token_value, type=token_type)
 
-        assert users_models.Token.query.filter_by(value=self.token_value).first() is not None
+        assert not token_utils.SixDigitsToken.token_exists(token_utils.TokenType.PHONE_VALIDATION, user.id)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Je ne comprend pas ce que tu test avec cette assertion.
La ligne en dessous doit suffire non ?

Reply:
je teste la même chose qui était testé auparavant. c'est qu'on a bien généré un Token. je n'ai pas remis en question l'utilité e celà

Reply:
Du coup faut retirer le `not` 
Là tu assert qu'on a _pas_ généré de token